### PR TITLE
edit examples docs page to recommend running in vscode and clarifying…

### PR DESCRIPTION
**Pull request by clmould**
… instructions for using jupyter

## Checklist

I confirm that I have completed the following checks:

- [ ] I have justified any large differences in the regression tests caused by this pull request in the comments.
- [ ] I have added new tests where appropriate for the changes I have made.
- [ ] If I have had to change any existing unit or integration tests, I have justified this change in the pull request comments.
- [ ] If I have made documentation changes, I have checked they render correctly.
- [ ] I have added documentation for my change, if appropriate.

### CODE REVIEW

File: documentation/proc-pages/usage/examples.md [hunk 1]
```diff
@@ -24,24 +49,6 @@ pip install notebook
...
-launch the server, navigate to the `examples` directory and run:
-
-```bash
-python -m notebook
-```
-
-A web browser will open and the notebook can be run from there. If you're using WSL, you'll 
-have to `ctrl + click` the link.
 
 <h2>Notebooks in Docker</h2>
```
Comment (timothy-nunn) [inline documentation/proc-pages/usage/examples.md:53]: This section is outdated now that we have removed the dockerfiles. Could you remove it for me in this PR too?

**Review by timothy-nunn — CHANGES_REQUESTED** [review]
Hi Clair, nothing wrong with what you have written. However, we should use this as an opportunity to refresh the rest of this page:
1. See my comment
2. Review this section https://github.com/ukaea/PROCESS/pull/3310/files#diff-b207a7cfdd17ae0611cf15fd03945225e1760a7ade29abbcc52d0dfe21f5c2b0R80-R87 to reflect the recent changes with testbook.

**Review by timothy-nunn — CHANGES_REQUESTED** [review]
You have removed the Docker section from `examples/README.md` but not `documentation/proc-pages/usage/examples.md`

File: documentation/proc-pages/usage/examples.md [hunk 2]
```diff
@@ -4,77 +4,49 @@ Examples of how to use PROCESS are provided in the form of Jupyter notebooks. Th
...
-apt-get update
-apt-get install poppler-utils
+pip install notebook
+``` 
+
+for just the `notebook` package.
+
 ```
 
 <h2>Maintaining notebooks and scripts</h2>
```
[the remark this answers is not in the document]
Comment (clmould) [reply documentation/proc-pages/usage/examples.md]: I replaced all <h2> with # now

**Review by timothy-nunn — APPROVED** [review]

**Pull request merged**